Spice/.env

```diff
@@ -0,0 +1,2 @@
+﻿STRIPE__SECRETKEY = {Key}
+STRIPE__PUBLISHABLEKEY = {Key} 
```

Comment:
Add you keys here, but don't commit this file. Add it to your .gitignore
E.g. STRIPE__SECRETKEY = sk_test_1234.....................